swagger.yaml

```diff
@@ -5613,6 +5618,31 @@ definitions:
         type: boolean
         description: Indication that the version is the recommended one.
 
+  openshift-images:
```

Comment:
openshift-images -> openshift-version-info

Reply:
hmm, do you think it reflects the object better? as it includes just the info about the images. Maybe openshift-images-info?:/

Reply:
I think image means two different things (docker image and ISO image) and version means something more unified in this case
anyway, just a nit